hieradata/dev/wso2/wso2is/pattern-2/default.yaml

```diff
@@ -296,3 +363,89 @@ wso2::cache:
   idp_cache_byhri : true
   idp_cache_by_name: true
 
+wso2::tomcat:
+ maxThreads : 500
+ acceptCount : 500
+
+# Secure vault configuration
```

Comment:
If the latest `wso2base` module is being used, these secure vault configs are not needed.